cdap-stream-clients/python/cdap_stream_client/streamclient.py

```diff
@@ -23,27 +23,22 @@
 
 class StreamClient(ConnectionErrorChecker):
 
-    __GATEWAY_VERSION = u'/v2'
     __REQUEST_PLACEHOLDERS = {
         u'streamid': u'<streamid>'
     }
-    __REQUESTS = {u'streams': __GATEWAY_VERSION + u'/streams'}
+    __REQUESTS = {u'streams': u'/streams'}
```

Comment:
I moved the version to the connection config, along with adding a namespace to the config. This is in parity with the Java stream client.